packages/grafana-schema/src/raw/composable/testdata/dataquery/x/TestDataDataQuery_types.gen.ts

```diff
@@ -11,7 +11,7 @@
 
 import * as common from '@grafana/schema';
 
-export const pluginVersion = "10.2.0-pre";
+export const pluginVersion = "%VERSION%";
```

Comment:
I'm struggling to understand if (or how) this gets replaced? 🤔

Reply:
tl;dr; it doesn't but it's not used

the code generator should have some logic that if a version is not set, it uses the Grafana version. Since the version is now defined it's using that value (which is `%VERSION%` now). In any case, this `pluginVersion` is not used so not something we should worry about.

Reply:
Ah ok thanks for the explanation I was somewhat confused. 👍